tests/dataplicity/subcommands/test_version.py

```diff
@@ -0,0 +1,13 @@
+from dataplicity.subcommands.version import Version
+from dataplicity import __version__
+
+
+def test_version(capsys):
```

Comment:
`capsys` ? Can we have explicit variable names please; underscores, no abbreviations...